splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -219,29 +236,66 @@ private void configureHandler() throws Exception {
                 .to(http("dynamic")
                         .httpMethod("POST")
                         .advanced()
-                        .httpClientConfigurer(getClientConfigurer()))
-                .endChoice()
+                        .httpClientConfigurer(getClientConfigurer(false)))
+                .log("Response ${body}")
+                .to(direct("success"))
                 .otherwise()
+                .when(simple("${headers.metadata[trustAll]} == 'true'"))
                 .to(https("dynamic")
                         .httpMethod("POST")
                         .advanced()
-                        .httpClientConfigurer(getClientConfigurer()))
-                .endChoice()
-                .end()
+                        .httpClientConfigurer(getClientConfigurer(true)))
+                .otherwise()
+                .to(https("dynamic")
+                        .httpMethod("POST")
+                        .advanced()
+                        .httpClientConfigurer(getClientConfigurer(false)))
                 // Log after a successful send.
                 .log("Response ${body}")
-                .to(direct("success"));
+                .to(direct("success"))
+                .endChoice()
+                .end();
     }
 
-    protected HttpClientConfigurer getClientConfigurer() {
+    protected HttpClientConfigurer getClientConfigurer(boolean trustAll) {
         return (clientBuilder) -> {
+
             // proactively evict expired connections from the connection pool using a background thread
             clientBuilder.evictExpiredConnections();
 
             // proactively evict idle connections from the connection pool after 5s using a background thread.
             // Arguments set maximum time persistent connections can stay idle while kept alive in the connection pool.
             // Connections whose inactivity period exceeds this value will get closed and evicted from the pool.
             clientBuilder.evictIdleConnections(5L, TimeUnit.SECONDS);
+
+            if (trustAll) {
+                // Heavily adapted from: https://gist.github.com/onderson/17bc7b41c0c28211639a5590a0051ae4
+                SSLContext sslContext = null;
+                try {
+                    sslContext = new SSLContextBuilder().loadTrustMaterial(null, new TrustStrategy() {
```

Comment:
Nit: There is a [lambda expression](https://www.w3schools.com/java/java_lambda.asp) syntax in Java now.
```
(X509Certificate[] arg0, String arg1) -> true
```